server/planning/content_api/events/module.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2025 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.core.resources import (
+    ResourceConfig,
+    MongoResourceConfig,
+    MongoIndexOptions,
+    ElasticResourceConfig,
+    RestEndpointConfig,
+)
+from content_api import MONGO_PREFIX, ELASTIC_PREFIX
+
+from .event import EventResourceModel
+from planning.events import EventsService
```

Comment:
We won't require the full EventsService from the base planning module. Instead we want a generic one without any app specifics (a simple data layer). Something as easy this should do:

```python
from superdesk.core.resources.service import AsyncResourceService

class ContentAPIEventService(AsyncResourceService[ContentAPIEventsResource]):
    pass
```